### PR TITLE
update enums article

### PR DESCRIPTION
**Pull request by danthe1st**
This PR contains minor updates to the enums article I originally contributed in #45.

I made changes to the writing (e.g. "exhaustiveness checking" instead of "exhaustiveness") and 
I made the following changes:
- use the plural of enum constants in a comment about listing them
- use the name "exhaustiveness checking" (should that be Title Case?) instead of just the concept of exhaustiveness as well as add a link to the patternmatching article (The article isn't available in this repository so I didn't use an `id:` link)
- move the enum constants of the enum with more than the enum declaration in the class body to two lines. This makes it easier to see the `;` after the enum declaration.
- add a few spaces in code as most people prefer that I think
- slightly reword the conclusion

### CODE REVIEW

File: app/pages/learn/01_tutorial/03_getting-to-know-the-language/04_classes_objects/01_enums.md [hunk 1]
```diff
@@ -74,8 +74,8 @@ switch (someDay) {
 With [Switch Expressions](id:lang.classes-objects.switch-expression),
 the compiler can check whether all values of the enum are handled.
 If any possible value is missing in a switch expression, there will be a compiler error.
-This is referred to as Exhaustiveness and can also be achieved with regular classes 
-through [Sealed Classes](https://openjdk.org/jeps/409).
+This is referred to as exhaustiveness checking and can also be achieved with regular classes
+through [Sealed Classes](https://openjdk.org/jeps/409) and [Patternmatching](/learn/pattern-matching/#switch).
```
Comment (ammbra) [inline app/pages/learn/01_tutorial/03_getting-to-know-the-language/04_classes_objects/01_enums.md:78]:
```suggestion
through [sealed classes](https://openjdk.org/jeps/409) and [pattern matching](/learn/pattern-matching/#switch).
```

File: app/pages/learn/01_tutorial/03_getting-to-know-the-language/04_classes_objects/01_enums.md [hunk 2]
```diff
@@ -74,8 +74,8 @@ switch (someDay) {
 With [Switch Expressions](id:lang.classes-objects.switch-expression),
```
Comment (ammbra) [inline app/pages/learn/01_tutorial/03_getting-to-know-the-language/04_classes_objects/01_enums.md:74]:
```suggestion
With [switch expressions](id:lang.classes-objects.switch-expression),
```

**Comment by danthe1st** [comment]
Thank you for your review. I addressed the feedback.

File: app/pages/learn/01_tutorial/03_getting-to-know-the-language/04_classes_objects/01_enums.md [hunk 3]
```diff
@@ -71,11 +71,11 @@ switch (someDay) {
 }
 ```
 
-With [Switch Expressions](id:lang.classes-objects.switch-expression),
+With [switch expressions](id:lang.classes-objects.switch-expression),
 the compiler can check whether all values of the enum are handled.
 If any possible value is missing in a switch expression, there will be a compiler error.
 This is referred to as exhaustiveness checking and can also be achieved with regular classes
-through [Sealed Classes](https://openjdk.org/jeps/409) and [Patternmatching](/learn/pattern-matching/#switch).
+through [sealed classes](https://openjdk.org/jeps/409) and [patternmatching](/learn/pattern-matching/#switch).
```
Comment (ammbra) [inline app/pages/learn/01_tutorial/03_getting-to-know-the-language/04_classes_objects/01_enums.md:78]:
```suggestion
through [sealed classes](https://openjdk.org/jeps/409) and [pattern matching](/learn/pattern-matching/#switch).
```
There is a space between pattern and matching 😁

**Comment by danthe1st** [comment]
Is there anything else I should change?

**Comment by ammbra** [comment]
All good.

**Review by ammbra — APPROVED** [review]
Looks good to me.

**Review by JosePaumard — APPROVED** [review]

**Pull request merged**